src/h5web/vis-packs/core/heatmap/hooks.ts

```diff
@@ -38,36 +31,3 @@ export function useTooltipFormatters(
       format('.3')(dataArray.get(ordinateToIndex(y), abscissaToIndex(x))),
   };
 }
-
-export function useImageSize(imageRatio: number | undefined): Size {
-  const { width, height } = useThree((state) => state.size);
-
-  if (!imageRatio) {
-    return { width, height };
-  }
-
-  const canvasRatio = width / height;
-  const mismatch = canvasRatio / imageRatio;
-
-  return {
-    width: mismatch > 1 ? height * imageRatio : width,
-    height: mismatch > 1 ? height : width / imageRatio,
-  };
-}
-
-export function useCanvasDomains(
-  imageSize: Size,
-  abscissaDomain: Domain,
-  ordinateDomain: Domain
-): [Domain, Domain] {
```

Comment:
Removed completely, as we no longer need to extend the domains -- we just let the unbound scales do their job.